.github/workflows/atqa-framework-on-checks-branch.yml

```diff
@@ -0,0 +1,202 @@
+name: 'Run update keeper tests'
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  update-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get sources with full history
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      #Set up master with tag
+
+      #      - name: Find current Git tag
+      #        run: echo ::set-output name=VALUE::$(git describe --abbrev=0 --tags `git rev-list --tags --max-count=1`)
+      #        id: currentTag
+      #
+      #      - name: Get sources for current tag
+      #        uses: actions/checkout@v3
+      #        with:
+      #          ref: '${{ steps.currentTag.outputs.VALUE }}'
+
+      - name: Find previous Git tag
+        run: echo ::set-output name=VALUE::$(git describe --abbrev=0 --tags `git rev-list --tags --skip=1 --max-count=1`)
+        id: prevTag
+
+      - name: Get sources for current tag
+        uses: actions/checkout@v3
+        with:
+          ref: '${{ steps.prevTag.outputs.VALUE }}'
```

Comment:
Step says "Get sources for current tag", but here it says `prevTag`. Is it current or previous?